### ISSUE TITLE
Getting timestamp from incoming message

### ISSUE THREAD

**Issue by drejc12** [gammu/gammu#618]
I am using gammu with Raspberry Pi and I would like to avoid spamming my program (RunOnReceive) from executing if it starts, but the system has some unread messages in the inbox from the time it was off.

I would like to run my bash script only when the message was sent to the system and avoid any reading from the inbox from before. I would like to throw away all messages that are >1min older than the current date/time.
In my script I use:
```
from=$SMS_1_NUMBER
message=$SMS_1_TEXT
```

Is there a way I can get a timestamp when the message _should_ be delivered with a $_____ command to access it? Or, is it even possible to get a timestamp? 

How could you avoid such spamming and avoid messages like that? Can there be anything else done on the code level?
Programming language: bash.

**Comment by berilslv** [comment]
yes, i would like to know if there is any variable which can be sent by sms, f.ex. timestamp
